lexer.html

```diff
@@ -0,0 +1,140 @@
+<link rel="import" href="common.html">
+<script>
+function ShadeyCssLexer() {
+  'use strict';
+  // NOTE(cdata): Generally, tokens are structured as three-valued tuple,
```

Comment:
I think it might be nicer to read token using code to have a token class/object. Created with a constructor, this won't be slower than a tuple-like array, and quite possibly will be faster.